.github/workflows/operator-ui.yml

```diff
@@ -32,29 +19,22 @@ jobs:
         with:
           basic-auth: ${{ secrets.GRAFANA_CLOUD_BASIC_AUTH }}
           hostname: ${{ secrets.GRAFANA_CLOUD_HOST }}
-          this-job-name: Operator UI Tests
+          this-job-name: Update Version 
         continue-on-error: true
-      - name: Setup node
-        uses: actions/setup-node@v2
-        with:
-          node-version: "16"
+
       - name: Checkout the repo
-        uses: actions/checkout@v2
-      - name: Cache dependencies
-        uses: actions/cache@v2
+        uses: actions/checkout@v3
+
+      - name: Update version
+        id: update
         env:
-          cache-name: yarn-cache
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: ./operator_ui/update.sh
+
+      - name: Open PR
+        uses: peter-evans/create-pull-request@18f90432bedd2afd6a825469ffd38aa24712a91d # v4.1.1
```

Comment:
Who is peter evans?

Reply:
OSS person that makes a lot of github interaction projects. I've pinned the git sha of external actions to avoid tag mutability. Github doesn't provide their own pull request action. We use it quite a bit around the org, see https://github.com/search?l=&q=peter-evans+user%3Asmartcontractkit&type=code for usage